target/linux/generic/files/drivers/net/phy/ar8327.c

```diff
@@ -680,7 +672,20 @@ ar8327_init_globals(struct ar8xxx_priv *priv)
 	for (i = 0; i < AR8XXX_NUM_PHYS; i++)
 		data->eee[i] = false;
 
+	if (chip_is_ar8327(priv))
+		ar8xxx_write(priv, AR8327_REG_GLOBAL_FC_THRESH,
+				AR8327_GLOBAL_FC_THRESH_DFLT_VAL);
+
 	if (chip_is_ar8337(priv)) {
+
+		/*
+		 * Workaround: RGMII RX delay setting needs to be
+		 * always specified for AR8337 to avoid port 5
+		 * RX hang on high traffic / flood conditions
+		 */
+		ar8xxx_write(priv, AR8327_REG_PAD5_MODE,
+			AR8327_PAD_RGMII_RXCLK_DELAY_EN);
```

Comment:
Did you really intend to overwrite the entire register here instead of just setting the extra bit?